pkg/controller/issueapi/issue.go

```diff
@@ -123,10 +139,19 @@ func (ic *IssueAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	code, err := codeRequest.Issue(ctx, ic.config.GetColissionRetryCount())
 	if err != nil {
 		ic.logger.Errorf("otp.GenerateCode: %v", err)
-		controller.WriteJSON(w, http.StatusOK, api.Error("error generating verification, wait a moment and try again"))
+		controller.WriteJSON(w, http.StatusInternalServerError, api.Error("error generating verification, wait a moment and try again"))
 		return
 	}
 
+	if smsProvider != nil {
+		message := fmt.Sprintf(smsTemplate, code, int(expiration.Minutes()))
+		if err := smsProvider.SendSMS(ctx, request.Phone, message); err != nil {
+			ic.logger.Errorf("otp.SendSMS: %v", err)
+			controller.WriteJSON(w, http.StatusInternalServerError, api.Error("failed to send sms: %v", err))
```

Comment:
Do we want to delete the code in that case?

Reply:
I think we want to save the code and present it on the screen for the clinician to read. WDYT?

Reply:
Done

Reply:
yeah, I think it's fine, but this only sends the error back